src/components/Carousel/Carousel.module.css

```diff
@@ -4,20 +4,22 @@
 
 .slides {
   position: relative;
-}
-
-.hasMultiple {
   cursor: e-resize;
 }
 
 .image {
   width: 100%;
   height: 100%;
+  overflow: hidden;
   opacity: 0;
 
-  &:not(:first-of-type) {
-    position: absolute;
```

Comment:
have to say, this feels a bit dirty. A position absolute image inside a position absolute div. Do we need the wrapper `div` at all?

Reply:
Yeah, good point, it was basically building on top of a pretty messy block of code with even more mess. Have taken the time to work out what is going on and simplify.